contracts/deploy/L2VRFConsumerMock/L2VRFConsumer.cjs

```diff
@@ -0,0 +1,18 @@
+const func = async function (hre) {
+  const { deployments, getNamedAccounts, network } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const registryDeployment = await deploy('L2VRFConsumerMock', {
+    args: ['0xccD917Bb5312d42260CD77c8DFc105293a37F9B5'],
```

Comment:
is L2Endpoint Address fixed into this value?
if there's other way to declare this address rather than magic number would be nice

Reply:
Roger that! Thank you